docs/examples/plot_ghi_transposition.py

```diff
@@ -0,0 +1,75 @@
+"""
+GHI to POA Transposition
+=========================
+
+Example of transposing clearsky GHI to POA
+"""
+
+# %%
+# This example shows how to use the get_clearsky method to generate clearsky
+# GHI data as well as how to use the get_total_iradiance function to transpose
+# GHI data to Plane of Array (POA) irradiance.
+
+from pvlib import location
+from pvlib.irradiance import get_total_irradiance
+import pandas as pd
+from matplotlib import pyplot as plt
+
+# For this example, we will be using Golden, Colorado
+tz = 'MST'
+lat, lon = 39.755, -105.221
+
+# Create location object to store lat, lon, timezone
+site = location.Location(lat, lon, tz=tz)
+
+
+# Define a function to handle the transposition
+def get_irradiance(site_location, date, tilt, surface_azimuth):
+    # Creates one day's worth of 10 min intervals
+    times = pd.date_range(date, freq='10min', periods=6*24, tz=tz)
+    # Generate cleaersky data using the Ineichen model, which is the default
+    # The get_clearsky method returns a dataframe with values for GHI, DNI,
+    # and DHI
+    clearsky_ghi = site_location.get_clearsky(times)
+    # Get solar azimuth and zenith to pass to the transposition function
+    solar_position = site_location.get_solarposition(times=times)
+    # Use the get_total_irradiance function to transpose the GHI to POA
+    POA_irradiance = get_total_irradiance(
+        surface_tilt=tilt,
+        surface_azimuth=surface_azimuth,
+        dni=clearsky_ghi['dni'],
+        ghi=clearsky_ghi['ghi'],
+        dhi=clearsky_ghi['dhi'],
+        solar_zenith=solar_position['zenith'],
```

Comment:
```suggestion
        solar_zenith=solar_position['apparent_zenith'],
```
FYI I'd like someone else to confirm that using apparent solar position is appropriate before making this change

Reply:
According to the parameters for each transposition model option in `get_total_irradiance`, `apparent_zenith` should be passed through, rather than `zenith`.

It's a different matter to verify that each model actually expects `apparent_zenith` rather than `zenith`. I'll look at the underlying references when I can find the opportunity.